### ISSUE TITLE
Database Navigator not displaying all columns 

### ISSUE THREAD

**Issue by enduser1604** [dbeaver/dbeaver#10703]
Hi @uslss 

Went to the source of the view I was having issues with and realized that the columns were erroring out(not showing in database navigator) in the source table as well. 

![javalang](https://user-images.githubusercontent.com/74217273/99729140-57efc580-2a6f-11eb-903b-d38bb19a75f1.png)

You can notice that the columns actually appear blank here, which is why the view has issues with displaying the columns in Database Navigator.

![stacktrace](https://user-images.githubusercontent.com/74217273/99729295-8b325480-2a6f-11eb-8eb8-20d7c1313f21.png)

Attached is the stacktrace for one of the errors.

[stacktrace.txt](https://github.com/dbeaver/dbeaver/files/5569987/stacktrace.txt)

DBeaver Version 7.2.3

Using PostgreSQL Redshift database using jdbc driver: RedshiftJDBC42-no-awssdk-1.2.x.x found here: https://docs.aws.amazon.com/redshift/latest/mgmt/configure-jdbc-connection.html

Also, a bit strange but in the blank fields, bottom left actually shows the column correctly? 
![invisible column](https://user-images.githubusercontent.com/74217273/99730582-8b335400-2a71-11eb-8135-a01e28cdc916.png)

_Originally posted by @enduser1604 in https://github.com/dbeaver/dbeaver/issues/10295#issuecomment-730663944_

**Comment by LonwoLonwo** [comment]
Hello @enduser1604 
Can you please test new [DBeaver versions](https://dbeaver.io/)? 
Some changes have already taken place since the release of 7.2.3
Thank you in advance

**Comment by LonwoLonwo** [comment]
I found it.
Was fixed here https://github.com/dbeaver/dbeaver/commit/0bb4852210d84c3f7304010dfe25581a3c74a717
There should be no errors in the current version

**Comment by enduser1604** [comment]
Hi @LonwoLonwo  and @uslss ,

I have tested out the issue in the new version. The underlying tables that were having issues are resolved (i.e. the ones that were producing the stacktrace posted above). Unfortunately,  when creating views on these tables, only a subset of columns are being returned and also are highlighted in red. Seems like just the bigint/numeric columns and the same issue originally is ocurring.

![dbeaver_issue](https://user-images.githubusercontent.com/74217273/102376756-cb0a3000-3f78-11eb-9f7a-7577bfa7b516.png)

Thanks in advance.

**Comment by LonwoLonwo** [comment]
@enduser1604 Could you please share with us the original table DDL? names are not important, column data types are important

**Comment by enduser1604** [comment]
@LonwoLonwo 

Below is the screenshot in database navigator of the source table that the view is based on. The view seems to omitt (string) columns. 
![dbeaver-ddl-strings](https://user-images.githubusercontent.com/74217273/102399422-77f2a600-3f95-11eb-92e2-3a60d0e35c14.png)



This is what SQL WB provides for the same table
![image](https://user-images.githubusercontent.com/74217273/102399154-19c5c300-3f95-11eb-9767-28dd264c30ff.png)


Perhaps is it because Redshift does not support string columns and utilizes varchar and text types? 

https://docs.aws.amazon.com/redshift/latest/dg/r_Character_types.html

**Comment by LonwoLonwo** [comment]
@enduser1604  Let's try to formulate the problem again. Some columns are missing in the view in the list of columns. And these columns have a certain type?
What about the "Source" tab for this view? Is DDL being discarded correctly?

**Comment by enduser1604** [comment]
@LonwoLonwo 
Your summary is accurate. Some columns are missing in the view and it appears to be only the columns of type "(string)". 

The view syntax is listed below: 
`
create view c_googleanalytics.web as
select * from trf_google_analytics.ga_prod
where partition_src = 'web'
with no schema binding;
`


Below is the table screenshot (trf_google_analytics.ga_prod): 
![image](https://user-images.githubusercontent.com/74217273/102401788-b3db3a80-3f98-11eb-88ff-24d685be02f4.png)


Below is the view screenshot ( c_googleanalytics.web) : 

![image](https://user-images.githubusercontent.com/74217273/102401850-c6ee0a80-3f98-11eb-864f-df83d89b7108.png)

**Comment by LonwoLonwo** [comment]
Data types "character" and "character varying" are now read correctly in the view.

**Comment by uslss** [comment]
verified

**Comment by LonwoLonwo** [comment]
@enduser1604 Hello. 
I fixed the problem. I hope I understood the problem correctly, because I could not reproduce the creation of a table with columns of type String. If you have the opportunity, please check the correctness of the work in our early version - https://dbeaver.io/files/ea/
It will be available tomorrow

**Comment by enduser1604** [comment]
@LonwoLonwo 

Issue has not been solved in 7.3.1 . As you can see, the db navigator does not show any columns in there except for ones that are bigints or booleans. The metadata after running a select *, showcases all the correct columns in the view. 

![image](https://user-images.githubusercontent.com/74217273/103161685-6fbb1780-479a-11eb-937e-e0cd661dfa9f.png)

**Comment by LonwoLonwo** [comment]
Hmm. Some questions, @enduser1604 
is this early version? https://dbeaver.io/files/ea/
please specify the specific version number. It can be found in the Help-About

![2020-12-28 12_33_20-About](https://user-images.githubusercontent.com/45152336/103205852-9482cc80-490b-11eb-9f0a-9b805e47f8c8.png)

**Comment by enduser1604** [comment]
@LonwoLonwo 

Thanks for pointing me to double check the version. I was not using the early access version when testing above. I have now tested early access via https://dbeaver.io/files/ea/  and it appears as if the issue has been fixed! 

Issue resolved: 
![image](https://user-images.githubusercontent.com/74217273/103447627-91a51600-4c42-11eb-8eb3-cf4c7ed58894.png)

Version that contains the fix/correct display:
![image](https://user-images.githubusercontent.com/74217273/103447639-a08bc880-4c42-11eb-870d-8471b8fdbfbc.png)


I will continue to QA for the next few days but it appears that this solved the main issue. I appreciate everyone's help.
